packages/node/src/login/oidc/oidcHandlers/RefreshTokenOidcHandler.ts

```diff
@@ -93,7 +94,7 @@ async function refreshAccess(
     };
     const authFetch = await buildAuthenticatedFetch(tokens.accessToken, {
       dpopKey,
-      refreshOptions: rotatedRefreshOptions,
+      refreshOptions: keepAlive ? rotatedRefreshOptions : undefined,
```

Comment:
This is the same as above, but it runs when logging a session in that has a refresh token. This is typically the flow used to reactivate a session pulled from storage: the user may not be present, so the auth code flow is not appropriate. If a refresh token is present, we go through the refresh flow, which results in a new access token to be issued.